index.js

```diff
@@ -1,6 +1,7 @@
 var onIdle = require('on-idle')
 var assert = require('assert')
 
+var disabled = window.localStorage.DISABLE_NANOTIMING === true
```

Comment:
oh lol oops, yeah this might blow up server rendering haha - great point.

Should do:
```js
typeof window !== 'undefined' && window.localStorage
```